--- a/target/linux/qualcommax/dts/ipq8072-ax9000.dts
+++ b/target/linux/qualcommax/dts/ipq8072-ax9000.dts
@@ -73,18 +73,21 @@
 		};
 
 		top-red {
+			label = "top:red";
 			gpios = <&tlmm 63 GPIO_ACTIVE_HIGH>;
 			color = <LED_COLOR_ID_RED>;
 			default-state = "keep";
 		};
 
 		top-green {
+			label = "top:green";
 			gpios = <&tlmm 64 GPIO_ACTIVE_HIGH>;
 			color = <LED_COLOR_ID_GREEN>;
 			default-state = "keep";
 		};
 
 		top-blue {
+			label = "top:blue";
 			gpios = <&tlmm 66 GPIO_ACTIVE_HIGH>;
 			color = <LED_COLOR_ID_BLUE>;
 			default-state = "keep";
